guide/src/config-languages.md

```diff
@@ -37,3 +38,4 @@ but block labels in the target language are not wanted or not supported, leave o
 | `block_start`       | Start sequence of block labels                                                                                                      |
 | `block_next`        | Start of next block with the same name                                                                                              |
 | `block_end`         | End of block labels                                                                                                                 |
+| `name_separator`    | Separator between block source file, name and index (e.g. `// <@main.rs#A clode block#0`)                                                  |
```

Comment:
If this necessarily part of a comment in the target language, couldn't Yarner select this automatically? The only requirement seems to be that it is not part of a code block name and even this could be worked around using escaping.

Reply:
That is how it was before. I thought the added flexibility would be better as when hardcoded it may collide with characters in names, or with block comment end sequences (for the rare case of languages that support no line comments).

Reply:
Hhhmmm, I think flexibility has a price mostly w.r.t. the mental energy required to make the abstract multitude of options concrete in ones mind, so I would also suggest weighing the cost against the benefits of added flexibility/configurability.

In this particular case, I think I would have stuck to '#' until someone actually has a problem with this choice.

Actually checking that block labels do not contain this character (where fixed or configurable) seems worthwhile though...

Reply:
> In this particular case, I think I would have stuck to '#' until someone actually has a problem with this choice.

Maybe this is the best way do go for now.